### DISCUSSION TITLE
Can't enable Copilot as part of Student Pack

### DISCUSSION THREAD

**Discussion by neeleshg23** [community/community#112638] [GitHub Education] [answered]
### Hi everyone,

Hi,

I can't enable GitHub Copilot despite completing the primer. I had paid for it in the past, so I fear that I can't get it for free since I've paid for it in the past. Help clarifying this would be greatly appreciated.

Thanks!

**Comment by samus-aran** [comment, marked as answer]
Hi @neeleshg23  👋

Ensure you have a successful GitHub Education [verification status](https://education.github.com/discount_requests/pack_application) and have received the confirmation email that your Academic benefits have been granted - this usually arrives 72 hours after you have been approved.

To enable Copilot, go to your account settings and under **Code, planning, and automation** section, select **Copilot**. You'll see "You're eligible to use copilot for free" and a button to claim the offer.

If you still unable to enable Copilot please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "I am a verified student having trouble redeeming my Global Campus offers". 👍

Hope this helps! Let us know how you get along 👍
